src/image-marker.js

```diff
@@ -0,0 +1,56 @@
+import { $ } from 'clappr'
+import StandardMarker from './standard-marker'
+
+/*
+ * An implementation of an image Marker, which can show an image.
+ */
+export default class ImageMarker extends StandardMarker {
+
+  /*
+   * time: the time in seconds that this marker represents
+   * tooltipImage: the image to be shown (optional)
+   * width: width for the image (Default: 200px)
+   * height: height for the image (Default: auto)
+   */
+  constructor (time, tooltipImage, width, height) {
+    super(time, tooltipImage)
+    this._tooltipText = tooltipImage || null
+    this._width = width || '200px'
```

Comment:
It can be a string but zepto (and jquery) will treat them as px values if it is a number, and I think javascript might work more efficiently with numbers. It's how I've seen other people do it and they do it in their example for font-size so that's what I'm going off.

It says in the [doc](http://zeptojs.com/#css)
> When a value for a property is blank (empty string, null, or undefined), that property is removed. When a unitless number value is given, “px” is appended to it for properties that require units.